### PR TITLE
make layout of json cleaner with consistent subobjects

### PR DESCRIPTION
**Pull request by manoj-freyr**
makign target and dtype part of json in action array results in consistent object structure making it easier to parse

### CODE REVIEW

File: tst.so/src/tst_worker.cpp [hunk 1]
```diff
@@ -319,7 +299,7 @@ bool TSTWorker::do_thermal_stress(void) {
         std::to_string(gpu_id) + " " + "max. junction temperature :" + " " + std::to_string(max_junction_temperature);
     rvs::lp::Log(msg, rvs::loginfo);
 
-    result = true;
+    //result = true;
```
Comment (jkottiku) [inline tst.so/src/tst_worker.cpp:302]: Any reason this is commented ?

File: tst.so/src/tst_worker.cpp [hunk 2]
```diff
@@ -120,8 +102,8 @@ void TSTWorker::log_interval_gflops(double gflops_interval) {
     string msg;
     msg = " GPU flops :" + std::to_string(gflops_interval);
     rvs::lp::Log(msg, rvs::logtrace);
-    log_to_json(TST_LOG_GFLOPS_INTERVAL_KEY, std::to_string(gflops_interval),
-                rvs::loginfo);
+    //log_to_json(TST_LOG_GFLOPS_INTERVAL_KEY, std::to_string(gflops_interval),
```
Comment (jkottiku) [inline tst.so/src/tst_worker.cpp:105]: Please remove commented code.

**Review by jkottiku — CHANGES_REQUESTED** [review]

File: tst.so/src/tst_worker.cpp (hunk 1 above)
Comment (manoj-freyr) [reply tst.so/src/tst_worker.cpp:302]: line is useless as we follow this with if else clause that appropriately updates result variable and result is anyway initialized to true

File: iet.so/src/iet_worker.cpp [hunk 3]
```diff
@@ -58,7 +58,7 @@
 #define IET_PWR_TARGET_ACHIEVED_MSG             "target achieved"
 #define IET_PWR_RAMP_EXCEEDED_MSG               "ramp time exceeded"
 #define IET_PASS_KEY                            "pass"
-
+#define RVS_TP_MESSAGE                          "target_power"
```
Comment (jkottiku) [inline iet.so/src/iet_worker.cpp:61]: Rename it to IET_JSON_LOG_TARGET_POWER_KEY ?

File: tst.so/src/tst_worker.cpp [hunk 4]
```diff
@@ -47,6 +47,9 @@
 #define TST_BLAS_FAILURE                        "BLAS setup failed!"
 #define TST_SGEMM_FAILURE                       "GPU failed to run the SGEMMs!"
 
+#define RVS_TT_MESSAGE                  "target_temp"
```
Comment (jkottiku) [inline tst.so/src/tst_worker.cpp:50]: Rename to TST_JSON_LOG_TARGET_TEMP_KEY & TST_JSON_LOG_THROTTLE_TEMP_KEY ?

**Review by jkottiku — APPROVED** [review]

File: tst.so/src/tst_worker.cpp (hunk 1 above)
Comment (jkottiku) [reply tst.so/src/tst_worker.cpp:300]: Ok. We can remove it then.

File: pbqt.so/src/action.cpp [hunk 5]
```diff
@@ -441,8 +431,9 @@ int pbqt_action::create_threads() {
         }
         if (0 != arr_linkinfo.size()) {
           /* Log link type */
-          log_json_data(std::to_string(srcnode), std::to_string(gpu_id[j]), rvs::logresults, 
-              pbqt_json_data_t::PBQT_LINK_TYPE, arr_linkinfo[0].strtype);
+          pbconn = arr_linkinfo[0].strtype;
+          //log_json_data(std::to_string(srcnode), std::to_string(gpu_id[j]), rvs::logresults,
```
Comment (jkottiku) [inline pbqt.so/src/action.cpp:435]: Remove commented code

**Review by jkottiku — APPROVED** [review]

**Pull request merged**